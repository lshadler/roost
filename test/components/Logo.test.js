import React from 'react';
import {render} from '@testing-library/react';

import Logo from '../../components/Logo';

describe('Logo Component', () => {
  test('renders with size=200px in default state', () => {
    const {asFragment} = render(<Logo />);

    expect(asFragment()).toMatchSnapshot();
  });

  test('renders with size=300px when size is overridden', () => {
    const {asFragment} = render(<Logo size="300px" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
