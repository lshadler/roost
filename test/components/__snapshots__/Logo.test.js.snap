// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component renders with size=200px in default state 1`] = `
<DocumentFragment>
  <svg
    data-name="Layer 1"
    id="Layer_1"
    preserveAspectRatio="true"
    viewBox="0 0 109.5 110.88"
    width="200px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <defs>
      <style>
        
                .cls-1 {
                    fill:#ffefc1;
                }
                
                .cls-1,.cls-2,.cls-3 {
                    stroke:#603813;
                    stroke-miterlimit:10;
                }
                
                .cls-2 {
                    fill:#a67c52;
                }
                
                .cls-3 {
                    fill:none;
                }
      </style>
    </defs>
    <ellipse
      class="cls-1"
      cx="57.07"
      cy="47.5"
      rx="31"
      ry="47"
    />
    <polyline
      class="cls-2"
      points="72.07 69.5 0.07 60.5 14.07 62.5 35.07 100.5 42.07 95.5 98.07 94.5 80.07 94.5 101.07 44.5 109.07 37.5 24.07 110.5"
    />
    <line
      class="cls-3"
      x1="94.07"
      x2="109.07"
      y1="50.5"
      y2="25.5"
    />
  </svg>
</DocumentFragment>
`;

exports[`Logo Component renders with size=300px when size is overridden 1`] = `
<DocumentFragment>
  <svg
    data-name="Layer 1"
    id="Layer_1"
    preserveAspectRatio="true"
    viewBox="0 0 109.5 110.88"
    width="300px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <defs>
      <style>
        
                .cls-1 {
                    fill:#ffefc1;
                }
                
                .cls-1,.cls-2,.cls-3 {
                    stroke:#603813;
                    stroke-miterlimit:10;
                }
                
                .cls-2 {
                    fill:#a67c52;
                }
                
                .cls-3 {
                    fill:none;
                }
      </style>
    </defs>
    <ellipse
      class="cls-1"
      cx="57.07"
      cy="47.5"
      rx="31"
      ry="47"
    />
    <polyline
      class="cls-2"
      points="72.07 69.5 0.07 60.5 14.07 62.5 35.07 100.5 42.07 95.5 98.07 94.5 80.07 94.5 101.07 44.5 109.07 37.5 24.07 110.5"
    />
    <line
      class="cls-3"
      x1="94.07"
      x2="109.07"
      y1="50.5"
      y2="25.5"
    />
  </svg>
</DocumentFragment>
`;
